tools/android/build_aar.py

```diff
@@ -198,6 +214,7 @@ def main():
     build_aar(
         output_dir=args.output_dir,
         config=args.config,
+        build_target=args.build_target,
```

Comment:
The nuget package uses the aar. Why do we need to ability to just build the shared library if that isn't needed for nuget?

Reply:
The pipeline before is:
```
x86   (all in sequential)
  |
x64
 |
armv7
 |
armv8
  |
aar
```
and Now
```
x86     x64   armv7   armv8  (in parallel)
\          \         /            /
      \           |         /
              aar
```

To support build sharelib in different platform  in parallel, this script has to be support build lib/aar/pack_only